src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.cs

```diff
@@ -37,8 +37,19 @@ internal static partial class ConflictResolver
 
         private const string s_metadataNameSeparators = " .,:<`>()\r\n";
 
-        internal static async Task<ConflictResolution> ResolveConflictsAsync(
-            RenameLocations renameLocationSet,
+        /// <summary>
+        /// Performs the renaming of the symbol in the solution, identifies renaming conflicts and automatically
+        /// resolves them where possible.
+        /// </summary>
+        /// <param name="replacementText">The new name of the identifier</param>
```

Comment:
Do we need to replace ISymbol usage here for `nonConflictSymbols` as well? Since this is already moved over to the LightweightRenameLocations